.gitignore

```diff
@@ -114,3 +114,6 @@ tags
 
 # End of https://www.gitignore.io/api/go,vim,emacs,visualstudiocode
 .idea
+
+# VSCode
+.vscode
```

Comment:
There's already a VSCode section - maybe we should just clean it up?

https://github.com/aslakknutsen/istio-workspace/blob/f8fe149b85bc476f7369442575b58a265afb6a27/.gitignore#L106-L112

Reply:
Fixed with fd578a1